packages/esm-generic-patient-widgets-app/package.json

```diff
@@ -36,7 +36,7 @@
   "dependencies": {
     "@carbon/charts-react": "^0.49.4",
     "@carbon/icons-react": "^10.18.0",
-    "@openmrs/esm-patient-common-lib": "^3.1.0",
+    "@openmrs/esm-patient-common-lib": "^3.2.0",
```

Comment:
Shouldn't this be a dev dependency and a peer dependency to treat the patient-common-lib as a singleton?